kolibri/core/assets/src/views/k-radio-button.vue

```diff
@@ -62,14 +65,21 @@
         required: true,
       },
       /**
-       * v-model value
+       * Description for Label
+       */
+      description: {
+        type: String,
+        required: false,
+      },
+      /**
+       * v-model value - the data that is currently assigned
        */
       value: {
         type: [String, Number, Boolean],
         required: true,
       },
       /**
-       * Unique value of the particular radio
+       * Unique value of the particular radio - the data that this button can assign
        */
       radiovalue: {
```

Comment:
`value` vs `radiovalue` is confusing IMO. 

Down the line, I'd like to use [Vue's `model` API](https://vuejs.org/v2/api/#model) to free up `value` to be used as `radiovalue` while maintaining `v-model` functionality.

Anyone have thoughts on this?

Reply:
seems reasonable to me, and agree that `value` vs `radiovalue` is confusing

@christianmemije do you recall if there was a specific motivation for the current setup?

Reply:
Can’t speak for Christian, but I have a feeling it had to do with wanted to be v-model compatible, which requires the value prop to hold the assigned value